compiler-rt/test/rtsan/blocking_call.cpp

```diff
@@ -28,7 +21,7 @@ int main() {
   return 0;
   // CHECK-NOT: {{.*safe_call*}}
   // CHECK: ==ERROR: RealtimeSanitizer: blocking-call
-  // CHECK-NEXT: Call to blocking function `custom_blocking_function` in real-time context!
+  // CHECK-NEXT: Call to blocking function `custom_blocking_function()` in real-time context!
```

Comment:
This change was because __func__ behaves different from the LLVM IR demangling of the same func